### PR TITLE
Add CVE-2024-33544

### PR DESCRIPTION
**Pull request by nmasdoufi-ol**

### CODE REVIEW

File: agent/exploits/cve_2024_33544.py [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
 
 import re
 
 from agent import definitions
 from agent import exploits_registry
 from agent.exploits import webexploit
 
-VULNERABILITY_TITLE = "SQL Injection in AA-Team WZone "
+VULNERABLE_VERSIONS = [
+    r"1[0-3](\.\d+){0,2}",  # Matches versions from 1.0 to 13.x.x
```
Comment (PiranhaSa) [inline agent/exploits/cve_2024_33544.py:10]: did you test this on something like 1.0 ?

Comment (nmasdoufi-ol) [reply agent/exploits/cve_2024_33544.py:10]: Yeah, it worked.

Comment (elyousfi5) [reply agent/exploits/cve_2024_33544.py]: can you add unit test for it please, i see only test for second regex

File: agent/exploits/cve_2024_33544.py [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
+class CVE202433544Exploit(webexploit.WebExploit):
+    accept_request = definitions.Request(
+        method="GET", path="/wp-content/plugins/wzone/readme.txt"
+    )
     check_request = definitions.Request(
-        method="GET", path="/cgi-bin/admin.cgi?Command=sysCommand&Cmd=ls"
+        method="GET", path="/wp-content/plugins/wzone/readme.txt"
     )
-    accept_pattern = [re.compile("TLR-2005KSH")]
+    accept_pattern = [re.compile("WZone")]
```
Comment (ybadaoui-ostorlab) [inline agent/exploits/cve_2024_33544.py:30]: Could you find another accept_pattern that is more precise?

Comment (nmasdoufi-ol) [reply agent/exploits/cve_2024_33544.py:30]: When visiting the readme file of a WP plugin, this is what you normally see:
```
=== WooCommerce ===
Contributors: automattic, woocommerce, mikejolley, jameskoster, claudiosanches, rodrigosprimo, peterfabian1000, vedjain, jamosova, obliviousharmony, konamiman, sadowski, wpmuguru, royho, barryhughes-1, claudiulodro, tiagonoronha, ryelle, levinmedia, aljullu, nerrad, joshuawold, assassinateur, haszari, mppfeiffer, nielslange, opr18, ralucastn, tjcafferkey, danielwrobert, patriciahillebrandt, albarin, dinhtungdu, imanish003, karolmanijak, sunyatasattva, alexandrelara, gigitux, danieldudzic, samueljseay, alexflorisca, opr18, tarunvijwani, pauloarromba, saadtarhi, bor0, kloon, coreymckrill, jorgeatorres, leifsinger
Tags: online store, ecommerce, shop, shopping cart, sell online
Requires at least: 6.5
Tested up to: 6.6
Requires PHP: 7.4
Stable tag: 9.1.2
License: GPLv3
License URI: https://www.gnu.org/licenses/gpl-3.0.html
```

So I guess the `accept_pattern` I used is fine.

**Review by 3asm — APPROVED** [review]

**Pull request merged**